### PR TITLE
Add support symbolic input/output for C

### PR DESCRIPTION
**Pull request by sava-cska**

### CODE REVIEW

**Review by ladisgin — CHANGES_REQUESTED** [review]

File: server/src/printers/KleePrinter.cpp [hunk 1]
```diff
@@ -32,6 +32,44 @@ printer::KleePrinter::KleePrinter(const types::TypesHandler *typesHandler,
...
+    std::string entryPoint = KleeUtils::entryPointFunction(tests, testMethod.name) + PrinterUtils::WRAPPED_SUFFIX;
+    strDeclareVar("int", KleeUtils::RESULT_VARIABLE_NAME, stringFunctionCall(entryPoint,
+                        {PrinterUtils::UTBOT_ARGC, PrinterUtils::UTBOT_ARGV, PrinterUtils::UTBOT_ENVP}));
+    strFunctionCall(PrinterUtils::POSIX_CHECK_STDIN_READ, {});
+    strReturn(KleeUtils::RESULT_VARIABLE_NAME);
+    closeBrackets(1);
+    ss << NL;
+}
+
+void KleePrinter::writeMain(const Tests &tests,
```
[the remark this answers is not in the document]
Comment (sava-cska) [reply server/src/printers/KleePrinter.cpp]: Rename

File: server/src/printers/KleePrinter.cpp [hunk 2]
```diff
@@ -121,11 +149,16 @@ fs::path KleePrinter::writeTmpKleeFile(
 }
 
 void KleePrinter::declTestEntryPoint(const Tests &tests,
-                                     const Tests::MethodDescription &testMethod) {
+                                     const Tests::MethodDescription &testMethod,
+                                     bool isWrapped) {
     std::string entryPoint = KleeUtils::entryPointFunction(tests, testMethod.name);
+    if (isWrapped) {
+        entryPoint += PrinterUtils::WRAPPED_SUFFIX;
```
[the remark this answers is not in the document]
Comment (sava-cska) [reply server/src/printers/KleePrinter.cpp]: Rename

**Review by vol0n — APPROVED** [review]

File: server/src/Tests.cpp [hunk 3]
```diff
@@ -512,6 +512,7 @@ void KTestObjectParser::parseKTest(const MethodKtests &batch,
         auto it = tests.methods.find<std::string, tests::Tests::MethodDescriptionToStringEqual>(
             testMethod.methodName);
         LOG_S(DEBUG) << "Parse klee for method: " << testMethod.methodName;
+        sourceFilePath = tests.sourceFilePath;
```
Comment (ladisgin) [inline server/src/Tests.cpp:515]: move from `for`

File: server/src/printers/Printer.h [hunk 4]
```diff
@@ -104,6 +104,9 @@ namespace printer {
                                SRef end = SCNL,
                                VSRef modifiers = {});
 
+        std::string stringFunctionCall(const std::string &functionName,
```
Comment (ladisgin) [inline server/src/printers/Printer.h:107]: strFunctionCall has same functionality

**Review by ladisgin — CHANGES_REQUESTED** [review]

File: server/src/printers/Printer.h (hunk 4 above)
Comment (sava-cska) [reply server/src/printers/Printer.h:107]: No

**Review by ladisgin — APPROVED** [review]

**Pull request merged**